tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test

```diff
@@ -0,0 +1,33 @@
+def hello_world(t):
+    """
+    :param t:xyz
+    :return:123"""
+    return 123
+
+class Documents(r):
+    """
+    Example Class
+    :param r:xyx
+    :return:215
+    """
+
+    def hello_Documents(self, z):
+        """
+        This is an inner function
+        :param z:xyz
+        :return:007
+        """
+        return 007
+    return 215
+
+class Hello(self)
+    """This is Hello class one liner docstring"""
+
+if 1 != 0:
+    def hello_planet(self):
```

Comment:
this is wrong.  this is not a method of the class `Hello`, so it shouldnt have `self`.  Your `if` is at the module level.